src/main/java/com/zendesk/maxwell/replication/BinlogConnectorEvent.java

```diff
@@ -62,7 +62,7 @@ public EventType getType() {
 
 	public void setLastHeartbeat(Long lastHeartbeat) {
 		this.position     = new BinlogPosition(gtidSetStr, gtid, this.position.getOffset(), this.position.getFile(), lastHeartbeat);
-		this.nextPosition = new BinlogPosition(gtidSetStr, gtid, this.position.getOffset(), this.position.getFile(), lastHeartbeat);
+		this.nextPosition = new BinlogPosition(gtidSetStr, gtid, this.nextPosition.getOffset(), this.nextPosition.getFile(), lastHeartbeat);
```

Comment:
This looks like it changes behaviour - the RowMap now ends up with `getPosition()` returning the _next_ position, not the position of the final event (which is used for marking a position as completed). Are we sure this is what we want?